doc/lua_api.txt

```diff
@@ -1340,6 +1340,12 @@ examples.
 #### `list[<inventory location>;<list name>;<X>,<Y>;<W>,<H>;<starting item index>]`
 * Show an inventory list
 
+#### `listring[<inventory location>;<list name>]`
+* Allows to create a ring of inventory lists
+* Shift-clicking on items in one element of the ring
+* will send them to the next inventory list inside the ring
+* The first occurence counts
```

Comment:
it doesnt exactly override, some people might intentionally specify an inventory list multiple times.